ui/partials/main/jobs.html

```diff
@@ -6,6 +6,12 @@
   <div class="progress-bar"  role="progressbar" style="width: 100%"></div>
 </div>
 
+<div uib-alert type="info" style="flex: none;" ng-show="serverChangedDelayed" ng-cloak>
```

Comment:
Just noticed that there's a bit of useless space at the bottom of this thing. Could you set padding-bottom to 0px here?